app/models/views/office_business_entity_state.rb

```diff
@@ -0,0 +1,54 @@
+module Views
+  class OfficeBusinessEntityState
```

Comment:
No, I meant that this class doesn't have a test, but I was just completely blind. My apologies!